lib/model/product.rb

```diff
@@ -4,7 +4,7 @@ class Product
 
     @@id = 0
 
-    def initialize(name:, price:)
+    def initialize(name, price)
```

Comment:
How is this model different from `Shop::Item`? You have keyword arguments in its initializer, but here you want to remove them.